util/lua/health_test.go

```diff
@@ -47,7 +47,7 @@ func TestLuaHealthScript(t *testing.T) {
 		require.NoError(t, err)
 		for i := range resourceTest.Tests {
 			test := resourceTest.Tests[i]
-			t.Run(test.InputPath, func(t *testing.T) {
+			t.Run(filepath.Join(strings.TrimPrefix(dir, "../../resource_customizations/"), test.InputPath), func(t *testing.T) {
```

Comment:
This adds the group/kind part to the test name, which really helps figure out which test is failing. The filenames aren't unique across groupkinds.